### PR TITLE
feature(exchange): package creation and banxa package migration

### PR DESCRIPTION
**Pull request by banklesss**
Related to [YOMO-1206](https://emurgo.atlassian.net/browse/YOMO-1206)

[YOMO-1206]: https://emurgo.atlassian.net/browse/YOMO-1206?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apps/wallet-mobile/src/features/RampOnOff/useCases/CreateExchange/CreateExchange.tsx [hunk 1]
```diff
@@ -1,5 +1,5 @@
-import {banxaModuleMaker} from '@yoroi/banxa'
-import {BanxaReferralUrlQueryStringParams} from '@yoroi/banxa/lib/typescript/translators/module'
+import {banxaModuleMaker} from '@yoroi/exchange'
+import {BanxaReferralUrlQueryStringParams} from '@yoroi/exchange/lib/typescript/translators/banxa/module'
```
Comment (stackchain) [inline apps/wallet-mobile/src/features/RampOnOff/useCases/CreateExchange/CreateExchange.tsx]: needs to be re-exported

Comment (banklesss) [reply apps/wallet-mobile/src/features/RampOnOff/useCases/CreateExchange/CreateExchange.tsx]: will have a look

File: packages/exchange/README.md
Comment (jorbuedo) [inline packages/exchange/README.md:1]: Should be exchange instead of transfer?

Comment (banklesss) [reply packages/exchange/README.md:1]: yeah! thanks

**Review by stackchain — APPROVED** [review]

**Pull request merged**